File: lsp/core/aws-lsp-core/src/credentials/resolveCredentialsRequest.ts
/**
 * Used to request credentials from the langauge server's host
 */
export interface ResolveCredentialsRequest {
    /**
     * Unique Id of request for IAM Credentials
     */
    requestId: string

    /**
     * When the request was produced, in milliseconds since Unix Epoch
     */
    issuedOn: number
}

File: lsp/client/visualStudio/IdesLspPoc/Credentials/ResolveCredentialsRequest.cs
namespace IdesLspPoc.Credentials
{
    /// <summary>
    /// Request for credentials from the langauge server
    /// </summary>
    public class ResolveCredentialsRequest
    {
        /// <summary>
        /// Unique Id of request for IAM Credentials
        /// </summary>
        public string RequestId;

        /// <summary>
        /// When the request was produced, in milliseconds since Unix Epoch
        /// </summary>
        public long IssuedOn;
    }
}